### ISSUE TITLE
Non-existing version number used to redirect to last / latest / current version

### ISSUE THREAD

**Issue by educhastenier** [bonitasoft/bonita-documentation-site#184]
Embedded Bonita Custom page examples have URL of type https://documentation.bonitasoft.com/bonita/**7.x**/layouts (example in resource https://github.com/bonitasoft/bonita-web-pages/blob/dev/uid-pages/page-application-home/src/ApplicationHomeBonita.json#L356)

More generally https://documentation.bonitasoft.com/bonita/**anything**/ used to redirect to latest version

**Comment by tbouffard** [comment]
This is not a regression (if we had decided to use this term in this project - which is not the case, let's avoid discussion about bug, regression, let's change things if they mess someone), this is a choice by design.

### usage in web-pages

For the link related to the web-pages code, this is something that has not been released yet, the latest 2021.1 community public release use a specific version: https://github.com/bonitasoft/bonita-web-pages/blob/7.12.1/uid-pages/page-application-home/src/ApplicationHomeBonita.json#L356
So please changes the links to `latest` in the `dev` branch instead of using non regular urls. During the latest months/years, an implicit consensus has emerged to use `latest` for such needs, so let's be consistent.
In addition, 7.x doesn't conform with the new Bonita versioning which follows `year.digit`: that also why the new site natively provides a `2021.1` version, `7.12` is redirected to this version.

I have also check bonitasoft internal code and the 7.12.2 version also uses specific versions
```json
      "value" : [
        "var htmlElements =  '<ul>' +\r",
        "                        '<li>' +\r",
        "                            '<a href=\"https://documentation.bonitasoft.com/bonita/7.9/layouts\" target=\"_blank\" class=\"text-info\">' + uiTranslate('Layout') + '</a>: ' +\r",
        "                            uiTranslate(\"the frame in which application pages are displayed.\") + \r",
        "                        '</li>' +\r",
        "\r",
        "                        '<li>' +\r",
        "                            '<a href=\"https://documentation.bonitasoft.com/bonita/7.9/pages\" target=\"_blank\" class=\"text-info\">' + uiTranslate('Pages') + '</a>: ' +\r",
        "                            uiTranslate('the user interface components of the application. An application must contain at least one page.') + \r",
        "                         '</li>' +\r",
        "\r",
        "                        '<li>' +\r",
        "                            '<a href=\"https://documentation.bonitasoft.com/bonita/7.5/processes\" target=\"_blank\" class=\"text-info\">' + uiTranslate('Processes') + '</a>: ' +\r",
        "                            uiTranslate('the logic execution components of the application. An application must contain at least one process.') + ' ' +\r",
        "                            uiTranslate('A process can contain') + ' ' + '<a href=\"https://documentation.bonitasoft.com/bonita/7.5/forms\" target=\"_blank\" class=\"text-info\">' + uiTranslate('forms') + '</a>, ' +\r",
        "                            uiTranslate('which are the user interface of the process.') +\r",
        "                        '</li>' +\r",
        "\r",
        "                        '<li>' +\r",
        "                            '<a href=\"https://documentation.bonitasoft.com/bonita/7.5/rest-api-extensions\" target=\"_blank\" class=\"text-info\">' + uiTranslate('REST API extensions') + '</a>: ' +\r",
        "                            uiTranslate('called from pages or forms to retrieve data to be presented to the user.') +\r",
        "                        '</li>' +\r",
        "                        \r",
        "                        '<li>' +\r",
        "                            '<a href=\"https://documentation.bonitasoft.com/bonita/7.5/themes\" target=\"_blank\" class=\"text-info\">' + uiTranslate('Theme') + '</a> : ' +\r",
        "                            uiTranslate('the appearance of application pages, including fonts and colors.') +\r",
        "                        '</li>' +\r",
        "                    '</ul>';\r",
        "\r",
        "return htmlElements;"
      ],
```

### the `anything` token

There is currently no plan to keep this behaviour which was a side effects of a bug in the former documentation site.
See #32

**Comment by tbouffard** [comment]
The work to do in in the https://github.com/bonitasoft/bonita-web-pages repository `dev` branch as mentioned in https://github.com/bonitasoft/bonita-documentation-site/issues/184#issuecomment-789087648
So closing as `won't fix`
